tests/test_pytest_cov.py

```diff
@@ -1916,6 +1918,7 @@ def find_labels(text, pattern):
 
 
 @pytest.mark.skipif("coverage.version_info < (5, 0)")
+@pytest.mark.skipif("coverage.version_info > (6, 4)")
```

Comment:
What's the reason for skipping coverage > 6.4?

Reply:
Newer output adds lines for test setup and teardown.